src/bosh_openstack_cpi/lib/cloud/openstack/cloud.rb

```diff
@@ -1,4 +1,5 @@
 require 'common/common'
+require 'membrane'
```

Comment:
otherwise there are many errors with `uninitialized constant`. Do you think I need to move it into https://github.com/cloudfoundry-incubator/bosh-openstack-cpi-release/blob/master/src/bosh_openstack_cpi/lib/cloud/openstack.rb ?